### PR TITLE
add new relic

### PR DESCRIPTION
**Pull request by rootedbox**
# Related Github tickets

- a list of github issues that are relevant for this PR

# Background

_A short paragraph on test what exactly this PR solves._

# Testing completed

- [ ] _a list of actions that you've performed to ensure that this PR works as intended_
- [ ] _99% of the times you should have an item: "wrote tests"_

# Breaking changes

- [ ] I have checked my code for breaking changes
- [ ] If there are breaking changes, there is a supporting migration.

### CODE REVIEW

**Comment by taariq** [comment]
Closing. we don't want a PR on master for this.

File: go.mod [hunk 1]
```diff
@@ -3,10 +3,9 @@ module github.com/palomachain/paloma
 go 1.19
 
 require (
-	github.com/CosmWasm/wasmd v0.30.0
```
Comment (rootedbox) [inline go.mod:6]: so revert these changes the `go get github.com/newrelic/go-agent/v3` did? @verabehr

Comment (verabehr) [reply go.mod:6]: Is it required to downgrade? If so, we may need to look further for an alternative solution, but if it is not required, we definitely don't want to downgrade any of these libraries. They're all core libraries.
